### PR TITLE
Api authorization

### PR DESCRIPTION
**Pull request by Adam-SungMin-Park**
Screen Shot 2021-01-26 at 16 25 07
Screen Shot 2021-01-26 at 16 25 23

### CODE REVIEW

File: api-authorization/server/authorization-middleware.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+   */
+
+  /**
+    * References:
+    * https://nodejs.org/api/http.html#http_message_headers
+    * https://github.com/auth0/node-jsonwebtoken#jwtverifytoken-secretorpublickey-options-callback
+    */
+
+  if (req.headers['x-access-token'] === undefined) {
+    res.status(401).json('Authentication required');
```
Comment (Zhiwutian) [inline api-authorization/server/authorization-middleware.js:23]: you should be throwing a client error here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: api-authorization/server/authorization-middleware.js (hunk 1 above)
Comment (Adam-SungMin-Park) [reply api-authorization/server/authorization-middleware.js]: Updated!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**